src/js/edu-benefits/5490/config/form.js

```diff
@@ -80,6 +85,104 @@ const formConfig = {
+        previousBenefits: {
+          title: 'Previous Benefits',
+          path: 'benefits-eligibility/previous-benefits',
+          initialData: {},
+          uiSchema: {
+            'ui:description': 'Prior to this application, have you ever applied for or received any of the following VA benefits?',
+            previousBenefits: {
+              // Add the view-only fields in the proper order
+              'ui:order': [
+                'disability',
+                'dic',
+                'chapter31',
+                'view:ownServiceBenefits',
+                'ownServiceBenefits',
+                'view:claimedSponsorService',
+                'claimedSponsorService'
+              ],
+              disability: {
+                'ui:title': 'Disability Compensation or Pension'
+              },
+              dic: {
+                'ui:title': 'Dependents\' Indemnity Compensation (DIC)'
+              },
+              chapter31: {
+                'ui:title': 'Vocational Rehabilitation benefits (Chapter 31)'
+              },
+              'view:ownServiceBenefits': {
+                'ui:title': 'Veterans education assistance based on your own service'
+              },
+              ownServiceBenefits: {
+                'ui:title': 'Specify benefits',
+                'ui:options': { expandUnder: 'view:ownServiceBenefits' }
+              },
+              'view:claimedSponsorService': {
+                'ui:title': 'Veterans education assistance based on someone else’s service.'
+              },
+              // Would this be better as a 'view:someNameHere' field? Would it
+              //  get flattened properly when sending to the api if so?
+              claimedSponsorService: {
+                'ui:options': { expandUnder: 'view:claimedSponsorService' },
+                chapter35: {
+                  'ui:title': 'Chapter 35 - Survivors’ and Dependents’ Educational Assistance Program (DEA)'
+                },
+                chapter33: {
+                  'ui:title': 'Chapter 33 - Post-9/11 GI Bill Marine Gunnery Sergeant David Fry Scholarship'
+                },
+                transferOfEntitlement: {
+                  'ui:title': 'Transferred Entitlement'
+                },
+                other: {
+                  'ui:title': 'Other benefit'
+                },
+                veteranFullName: _.merge(fullName.uiSchema, {
+                  'ui:title': 'Sponsor Veteran’s name'
+                }),
+                veteranSocialSecurityNumber: _.merge(ssn.uiSchema, {
+                  'ui:title': 'Sponsor SSN'
+                })
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              previousBenefits: (() => {
+                // Not sure how to do this all in one fell swoop
+                // Also not conviced this is the best way to do this either
+                const newSchema = _.merge(previousBenefits, {
```

Comment:
Is there a better way to do this? I'm not a fan of artificially creating another object just to get `expandUnder` to group them all together, then flattening the object in `transform`.